src/components/AlertWarning/index.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+// import WarningIcon from 'components/Icon/WarningIcon';
```

Comment:
Please remove these commented lines 🙏

Reply:
Removed in 6b86964c2e1dc46a78855c6db1d2228c0c0e2885